downloader/history_downloader/history_manager.py

```diff
@@ -0,0 +1,19 @@
+import ccxt
+
+
+class HistoryDownloader:
+
+    def __init__(self, exchange_name: str, crypto_symbol: str):
+        """ init
+            Paramaters
+            ------------
+            exchange_name: str required - name of cryptocurrency, e.g. 'binance'

```

Comment:
A one simple suggestion, docs generator will take parametres description like this:
```suggestion
            exchange_name: str required
                name of cryptocurrency, e.g. 'binance'

```